### ISSUE TITLE
Tagging errors for "much"

### ISSUE THREAD

**Issue by muchang** [UniversalDependencies/UD_English-GUM#20]
For the following sentences, the words `much` seem to be `ADV`, while in the corpus, it is tagged as `ADJ`:
```
# sent_id = GUM_interview_herrick-38
# s_type = wh
# speaker = JackHarris
# text = How much more could you accomplish as an individual if you could learn any activity?
1	How	how	ADV	WRB	PronType=Int	2	advmod	2:advmod	Discourse=justify:85->80
2	much	much	ADJ	JJ	Degree=Pos	3	amod	3:amod	_
3	more	more	ADJ	JJR	Degree=Cmp	6	obj	6:obj	_
4	could	could	AUX	MD	VerbForm=Fin	6	aux	6:aux	_
5	you	you	PRON	PRP	Case=Nom|Number=Sing|Person=2|PronType=Prs	6	nsubj	6:nsubj	Entity=(person-80)
6	accomplish	accomplish	VERB	VB	VerbForm=Inf	0	root	0:root	_
7	as	as	ADP	IN	_	9	case	9:case	_
8	an	a	DET	DT	Definite=Ind|PronType=Art	9	det	9:det	_
9	individual	individual	NOUN	NN	Number=Sing	6	obl	6:obl:as	_
10	if	if	SCONJ	IN	_	13	mark	13:mark	Discourse=condition:86->85
11	you	you	PRON	PRP	Case=Nom|Number=Sing|Person=2|PronType=Prs	13	nsubj	13:nsubj	Entity=(person-80)
12	could	could	AUX	MD	VerbForm=Fin	13	aux	13:aux	_
13	learn	learn	VERB	VB	VerbForm=Inf	6	advcl	6:advcl:if	_
14	any	any	DET	DT	_	15	det	15:det	Entity=(event-85
15	activity	activity	NOUN	NN	Number=Sing	13	obj	13:obj	Entity=event-85)|SpaceAfter=No
16	?	?	PUNCT	.	_	6	punct	6:punct	_
```

```
# sent_id = GUM_fiction_pag-42
# s_type = decl
# text = Every child knows that much instinctively.
1	Every	every	DET	DT	_	2	det	2:det	Discourse=justify:81->80|Entity=(person-79
2	child	child	NOUN	NN	Number=Sing	3	nsubj	3:nsubj	Entity=person-79)
3	knows	know	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
4	that	that	DET	DT	Number=Sing|PronType=Dem	5	det	5:det	Entity=(event-75
5	much	much	ADJ	JJ	Degree=Pos	3	obj	3:obj	Entity=event-75)
6	instinctively	instinctively	ADV	RB	Degree=Pos	3	advmod	3:advmod	SpaceAfter=No
7	.	.	PUNCT	.	_	3	punct	3:punct	_
```
Commit: d38df82

**Comment by nschneid** [comment]
"much" in the first sentence and "that" in the second sentence should be ADV/advmod.

I'm not sure about "much" in the second sentence—it is a standalone quantifier meaning approximately "that much stuff/information".

**Comment by muchang** [comment]
Yes, I agree with your interpretation of the second sentence. It seems to omit the object.

**Comment by amir-zeldes** [comment]
OK, thanks - fixed in amir-zeldes/gum@6a8c0b6
